pkg/crc/network/proxy.go

```diff
@@ -200,9 +200,14 @@ func (p *ProxyConfig) tlsConfig() *tls.Config {
 	if p.ProxyCACert == "" {
 		return nil
 	}
-	caCertPool := x509.NewCertPool()
-	caCertPool.AppendCertsFromPEM([]byte(p.ProxyCACert))
-
+	caCertPool, err := x509.SystemCertPool()
+	if err != nil {
```

Comment:
If we are not sending error to upper layer (caller function) so better to put the error in debug log so we know what caused this error.

Reply:
will take of this in some following PRs.

Reply:
Should be fine now. `proxy.go` did not include `logging`, this is the main/bad reason for not adding logging in the first place.